installation/helm-chart/templates/KAFKA/kafka-consumer.yaml

```diff
@@ -48,60 +54,30 @@ spec:
       labels:
         app: kafka-consumer
     spec:
+      {{- if .Values.imagePullSecrets }}
       {{- with .Values.imagePullSecrets }}
       imagePullSecrets:
         {{- toYaml . | nindent 8 }}
       {{- end }}
-      containers:
-      - name: consumer
-        image: {{ .Values.registryHost }}/estela-consumer
-        imagePullPolicy: Always
-        command: ['python', 'consumer.py', 'job_requests']
-        env:
-        - name: KAFKA_ADVERTISED_PORT
-          value: "9092"
-        - name: KAFKA_ADVERTISED_LISTENERS
-          value: kafka
-        - name: PRODUCTION
-          value: "False"
-        - name: MONGO_CONNECTION
-          valueFrom:
-            secretKeyRef:
-              name: kafka-secrets
-              key: MONGO_CONNECTION
----
-kind: Deployment
-apiVersion: apps/v1
-metadata:
-  namespace: default
-  name: kafka-logs-consumer
```

Comment:
Why are we removing the logs consumer?

Reply:
@rodp63 The `kafka-logs-consumer` is currently consuming the Scrapy stats of jobs. What's missing is that it consumes all logs. We should not remove it.

Reply:
Got it